packages/path_provider/path_provider_windows/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This requires a version change, as described in the link from the relevant PR checklist item.